src/main/java/stsjorbsmod/relics/BottledBurdenRelic.java

```diff
@@ -0,0 +1,139 @@
+            }
+        }
+    }
+
+    @Override
+    public void onEquip() {
+        // follow same behavior as base game bottle relics.
+        CardGroup nonExertCards = CardUtils.getCardsForBottling(AbstractDungeon.player.masterDeck.getPurgeableCards());
+        nonExertCards.group.removeIf(c -> c.hasTag(HAS_EXERT));
+        if (nonExertCards.size() > 0) {
+            this.cardSelected = false;
+            if (AbstractDungeon.isScreenUp) {
+                AbstractDungeon.dynamicBanner.hide();
+                AbstractDungeon.overlayMenu.cancelButton.hide();
+                AbstractDungeon.previousScreen = AbstractDungeon.screen;
+            }
+
+            AbstractDungeon.getCurrRoom().phase = AbstractRoom.RoomPhase.INCOMPLETE;
+            AbstractDungeon.gridSelectScreen.open(nonExertCards, 1, this.DESCRIPTIONS[1] + this.name + LocalizedStrings.PERIOD, false, false, false, false);
+        }
+    }
+    @Override
+    public void onUnequip() {
+        if (this.card != null) {
+            AbstractCard cardInDeck = AbstractDungeon.player.masterDeck.getSpecificCard(this.card);
+            if (cardInDeck != null) {
+                inBottledBurden.set(card, false);
+            }
+        }
+    }
+
+    @Override
+    public void update() {
+        super.update();
+        if (!this.cardSelected && !AbstractDungeon.gridSelectScreen.selectedCards.isEmpty()) {
+            this.cardSelected = true;
+            this.card = AbstractDungeon.gridSelectScreen.selectedCards.get(0);
+            inBottledBurden.set(card, true);
+            AbstractDungeon.getCurrRoom().phase = AbstractRoom.RoomPhase.COMPLETE;
+            AbstractDungeon.gridSelectScreen.selectedCards.clear();
+            this.description = this.DESCRIPTIONS[2] + FontHelper.colorString(this.card.name, "y") + this.DESCRIPTIONS[3];
+            this.tips.clear();
+            this.tips.add(new PowerTip(this.name, this.description));
+            this.initializeTips();
+
+            AbstractCard cardInDeck = AbstractDungeon.player.masterDeck.getSpecificCard(this.card);
+            EntombedField.entombed.set(cardInDeck, true);
+            SelfExhumeFields.selfExhumeOnTurnX.set(cardInDeck, true);
+        }
+    }
+
+    public void setDescriptionAfterLoading() {
+        this.description = this.DESCRIPTIONS[2] + FontHelper.colorString(this.card.name, "y") + this.DESCRIPTIONS[3];
+        this.tips.clear();
+        this.tips.add(new PowerTip(this.name, this.description));
+        this.initializeTips();
+    }
+
+    @Override
+    public void atBattleStart() {
+        AbstractPower exhumeOnTurnXPower = new ExhumeOnTurnXPower(AbstractDungeon.player, AbstractDungeon.player.masterDeck.getSpecificCard(this.card), EXHUME_TURN);
+        //addToTop(new ExhaustSpecificCardAction(this.card, AbstractDungeon.player.drawPile));
+        addToTop(new ApplyPowerAction(AbstractDungeon.player, AbstractDungeon.player, exhumeOnTurnXPower));
```

Comment:
I would recommend having the relic itself implement an `atTurnStart` rather than delegating that to an `InvisiblePower`, and omit the new `InvisiblePower` altogether; in general, we try to avoid `InvisiblePower` because they tend to interact with effects that target "buffs" or "debuffs" (eg, the new Amnesia curse, or similar effects from other mods) in ways that are confusing to the player